### PR TITLE
Improve file upload UI (version #2; commits cleaned up)

### PR DESCRIPTION
**Pull request by weedySeaDragon**
PT Story: Need to improve file-upload visibility
https://www.pivotaltracker.com/story/show/141029197

This is a first attempt at improving the user experience (UX) when a user fills out a membership application.  Users are skipping over the section to upload files.

**Translations needed!**

(This replaces PR #188 because that one had _merges_ in it that added files not part of this PR.)

### Changes proposed in this pull request:
1.  Add instructions so users understand the importance of uploading files. Let users know they'll have 3 sections to complete.
2.  Set off each of the 3 major sections so users can see each of them clearly. (Indent the text under each section header to help set them off.)
3. If a users selects more than 1 file to upload, show the list of the files that will be uploaded when they submit the page.
3. Use background color and styling per @thesuss example in PivT to highlight the file upload section even more
4. If no files have been uploaded, show the warning icon
5. Added CSS classes and ids to views so styles can be applied

## Ready for Review
@patmbolger @thesuss 

### If no files have been uploaded:

**New/edit view:**

new-app-nofiles-warning-sv

---

### Applicant has chosen 3 files but hasn't saved their new application yet:

new-app-nofiles-warning-3-to-update-sv

### Editing an application with files already uploaded:

edit-app-with-2files-sv

---

(English):

edit-app-with-2files-en

---

### Viewing an application

show-application-sv

Ready for review:

### CODE REVIEW

File: config/locales/sv.yml [hunk 1]
```diff
@@ -163,6 +163,10 @@ sv:
   membership_applications:
     new:
       title: Ansök om medlemskap
+      instructions: some instructions here. 3 main sections. explain importance of the uploaded documents
```
Comment (thesuss) [inline config/locales/sv.yml]: För att kunna behandla din ansökan behöver vi ha information om dig, vilka kategorier du ansöker som samt dokument som styrker din kompetens i dessa kategorier.

File: config/locales/en.yml [hunk 2]
```diff
@@ -163,6 +163,10 @@ en:
   membership_applications:
     new:
       title: Apply for a Membership
+      instructions: some instructions here. 3 main sections. explain importance of the uploaded documents
```
Comment (thesuss) [inline config/locales/en.yml]: To process your application, we need information about you and in which category you are applying as well as documents proving your skills in this/those categories.

File: config/locales/sv.yml [hunk 3]
```diff
@@ -173,10 +177,15 @@ sv:
       contact_email: *contact_email
 
       can_be_same_email: Kan vara samma som du loggar in med, men behöver inte vara det.
-      application_dog_category: &application_dog_category "Ansöker som hund-*"
+      application_dog_category: &application_dog_category "Ansöker som hund-"
       clue_selected_categories: &clue_selected_categories "Blå fält = valda kategorier"
-      upload_a_file: 'Ladda upp en fil:'
+
+      upload_files_section: Underlag för att styrka min ansökan
+      upload_files_instructions: Why these are really important... SHF uses these to verify your certifications and methods. ... This ensures the standards remain high, which means the public will know that the SHF H-Marke is [valuable, credible, respected, worthy..].  (More words needed here.)
```
Comment (thesuss) [inline config/locales/sv.yml]: Dokumenten är mycket viktiga för att vi ska kunna bedöma din ansökan. Vi använder dem för att verifiera dina kunskaper och utbildningar. Detta säkerställer att vår medlemsstandard är fortsatt hög, vilket i förlängningen innebär att allmänheten vet att Sveriges hundföretagare är professionella och att lita på.

File: config/locales/sv.yml [hunk 4]
```diff
@@ -173,10 +177,15 @@ sv:
...
+      upload_files: 'Ladda upp filer från datorn'
       upload_more_files: Du kan ladda upp fler filer genom att redigera din ansökan efter att du skickat in den.
+      upload_button: Välj en fil...
+      will_be_uploaded: 'These files will be uploaded when you submit your application:'
```
Comment (thesuss) [inline config/locales/sv.yml]: Dessa filer kommer att laddas upp när du skickar in din ansökan:

**Review by thesuss — APPROVED** [review]

**Review by patmbolger — APPROVED** [review]

**Pull request merged**